pt-BR/mail/chrome/messenger/messenger.properties

```diff
@@ -485,426 +485,426 @@ copyToFolderAgain=Copiar de novo para “%1$S”
+pbSearchThresholdNotifcationBar.folder.title=Talvez os resultados da pesquisa estejam incompletos.
+pbSearchThresholdNotifcationBarResumeIndex.folder.label=Indexar esta pasta...
 pbSearchThresholdNotifcationBarResumeIndex.folder.accesskey=I
 
-pbSearchThresholdNotifcationBar.remind.label=Remind Me Later
+pbSearchThresholdNotifcationBar.remind.label=Lembrar mais tarde
 pbSearchThresholdNotifcationBar.remind.accesskey=R
 
-analyticsNotification.title=Help make Postbox better by automatically sending anonymous usage statistics.
-analyticsNotificationYes.label=Yes, I'll help!
+analyticsNotification.title=Ajude a melhorar o Postbox enviando automaticamente estatísticas de uso de forma anônima.
+analyticsNotificationYes.label=Sim, eu ajudarei!
 analyticsNotificationYes.accessKey=Y
-analyticsNotificationNo.label=No, thanks.
+analyticsNotificationNo.label=Não, obrigado.
 analyticsNotificationNo.accessKey=N
-analyticsNotificationLearnMore.label=Learn More...
+analyticsNotificationLearnMore.label=Saiba mais...
 analyticsNotificationLearnMore.accessKey=L
 
-facebookNotification.title=Log in to Facebook to see photos and status updates.
-facebookNotificationLogin.label=Log in
+facebookNotification.title=Faça o login no Facebook para ver fotos e atualizações de status.
+facebookNotificationLogin.label=Fazer login
 facebookNotificationLogin.accessKey=L
-facebookNotificationNever.label=No, thanks.
+facebookNotificationNever.label=Não, obrigado.
 facebookNotificationNever.accessKey=N
 
-twitterNotification.title=Log in to Twitter to see profile photos from the people you follow.
-twitterNotificationLogin.label=Log in
+twitterNotification.title=Faça o login no Twitter para ver fotos dos perfis das pessoas que você segue.
+twitterNotificationLogin.label=Fazer login
 twitterNotificationLogin.accessKey=L
-twitterNotificationNever.label=No, thanks.
+twitterNotificationNever.label=Não, obrigado.
 twitterNotificationNever.accessKey=N
 
-linkedinNotification.title=Log in to LinkedIn to match contacts with LinkedIn profile data and photos.
-linkedinNotificationLogin.label=Log in
+linkedinNotification.title=Faça o login no LinkedIn para fazer a correspondência de contatos com as informações e fotos dos perfis do LinkedIn.
+linkedinNotificationLogin.label=Fazer login
 linkedinNotificationLogin.accessKey=L
-linkedinNotificationNever.label=No, thanks.
+linkedinNotificationNever.label=Não, obrigado.
 linkedinNotificationNever.accessKey=N
 
-trialExpiringDaysNotification.title=This demo version of Postbox is going to expire in %1$S days.
-trialExpiringDayNotification.title=This demo version of Postbox is going to expire in %1$S day.
-trialExpiringTodayNotification.title=This demo version of Postbox is going to expire today.
-trialExpiredNotification.title=This demo version of Postbox has expired.
-trialExpiringNotificationUpgrade.label=Purchase Postbox...
+trialExpiringDaysNotification.title=Esta versão de demonstração do Postbox vai expirar em %1$S dia.
```

Comment:
CORRECTIONS NEEDED it should be trialExpiringDaysNotification.title=Esta versão de demonstração do Postbox vai expirar em %1$S dias.